apps/teamsjs-diagnostic-app/.gitignore

```diff
@@ -0,0 +1,18 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+# TeamsFx files
+env/.env.*.user
```

Comment:
Can we merge this `.gitignore` to the root repository's `.gitignore` or would that cause issues?